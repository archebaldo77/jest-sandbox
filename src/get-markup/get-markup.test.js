const getMarkup = require(`./get-markup`);

test(`Snapshot test for getMarkup (fn)`, () => {
  expect(getMarkup()).toMatchSnapshot();
});
