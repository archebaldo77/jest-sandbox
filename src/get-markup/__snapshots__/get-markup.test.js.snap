// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Snapshot test for getMarkup (fn) 1`] = `"<div class="welcome">Hello world!</div>"`;
